flashinfer/gemm/kernels/cutile/ragged_block_scaled_bmm_cutile.py

```
@@ -0,0 +1,613 @@
+    assert b_scale.is_contiguous(), "B scale matrix must be contiguous"
+    assert m_indptr.is_contiguous(), "m_indptr must be contiguous"
+
+    # Get dimensions
+    total_m, K_A = a.shape
+    Q, N, K_B = b.shape
+
+    assert K_A == K_B, f"K dimensions must match: {K_A} != {K_B}"
+    assert m_indptr.shape[0] == Q + 1, "m_indptr must have Q+1 elements"
+
+    # Validate scale dimensions
+    Q_SB, rnb, rkb = b_scale.shape
```

Ruff (0.15.21)

[warning] Unpacked variable `rnb` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)